### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by git-0r**
# Created ecom auth screens
# 1. Login
ecom7
# 2. Log out
ecom8
# 3. Signup
ecom9

### CODE REVIEW

File: css/cart_page.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/cart_page.css]: use rem

File: css/cart_page.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/cart_page.css]: use rem

File: css/cart_page.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  border: 1px solid var(--clr-black-mid);
+}
+
+.payment-info .order-btn {
+  width: 100%;
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (manish-GH) [inline css/cart_page.css]: use rem

File: css/cart_page.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  background-color: var(--clr-white-disabled);
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline css/cart_page.css]: use em for media queries

File: css/listing_page.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/listing_page.css]: use rem

File: css/listing_page.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/listing_page.css]: use rem

File: css/listing_page.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.product-listing-navigation {
+  width: 100%;
+  padding: 2rem;
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (manish-GH) [inline css/listing_page.css]: use rem

File: css/listing_page.css [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+  background-color: var(--clr-white-disabled);
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline css/listing_page.css]: use em

File: css/login_page.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/login_page.css]: use rem

File: css/login_page.css [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/login_page.css]: use rem

File: css/logout_page.css [hunk 11]
```diff
@@ -0,0 +1,58 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/logout_page.css]: use rem

File: css/logout_page.css [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/logout_page.css]: use rem

File: css/logout_page.css [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  flex-direction: column;
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline css/logout_page.css]: use em

File: css/signup_page.css [hunk 14]
```diff
@@ -0,0 +1,64 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/signup_page.css]: use rem

File: css/signup_page.css [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/signup_page.css]: use rem

File: css/signup_page.css [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+.login-form .login-btn {
+  width: 100%;
+}
+
+.login-form .signup-btn {
+  display: block;
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline css/signup_page.css]: use em

File: css/wishlist_page.css [hunk 17]
```diff
@@ -0,0 +1,100 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline css/wishlist_page.css]: use rem

File: css/wishlist_page.css [hunk 18]
```diff
@@ -0,0 +1,100 @@
...
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline css/wishlist_page.css]: use rem

File: css/wishlist_page.css [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.wishlist-header {
+  margin: 1rem 2rem;
+  text-decoration: underline;
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (manish-GH) [inline css/wishlist_page.css]: use rem

File: css/wishlist_page.css [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+  gap: 1rem;
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline css/wishlist_page.css]: use em

File: index.css [hunk 21]
```diff
@@ -0,0 +1,97 @@
+body {
+  max-width: 1440px;
```
Comment (manish-GH) [inline index.css]: use rem

File: index.css [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+  margin: 0 auto;
+}
+
+.top-navbar {
+  padding: 1rem;
+  max-height: 60px;
```
Comment (manish-GH) [inline index.css]: use rem

File: index.css [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+  width: 100%;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (manish-GH) [inline index.css]: use rem

File: index.css [hunk 24]
```diff
@@ -0,0 +1,97 @@
...
+header {
+  margin: 2rem;
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline index.css]: use em

File: pages/login_page.html [hunk 25]
```diff
@@ -0,0 +1,85 @@
...
+                </ul>
+            </div>
+            <div class="d-flex flex-justify-between">
+                <div>
+                    <input type="checkbox" name="remember" id="remember">
+                    <label for="remember" class="text-sm">Remember me</label>
+                </div>
+                <p class="text-sm">forgot password?</p>
+            </div>
+            <button class="btn btn-primary login-btn">Submit</button>
```
Comment (manish-GH) [inline pages/login_page.html]: replace Submit with Login

**Review by manish-GH — APPROVED** [review]
The UI is awesome and responsive too, Nice work. 

Add hover effect for your buttons.
Replace px with rem and em wherever mentioned and you are good to merge

File: pages/login_page.html [hunk 26]
```diff
@@ -0,0 +1,85 @@
...
+                    <li>one number</li>
+                    <li>minimum length 8 characters</li>
+                </ul>
+            </div>
+            <div class="d-flex flex-justify-between">
+                <div>
+                    <input type="checkbox" name="remember" id="remember">
+                    <label for="remember" class="text-sm">Remember me</label>
+                </div>
+                <p class="text-sm">forgot password?</p>
```
Comment (Naman-Saxena1) [inline pages/login_page.html]: Forgot password as link would be better

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: pages/login_page.html (hunk 26 above)
Comment (Sahil25061999) [inline pages/login_page.html]: use link for forgot password

**Review by Sahil25061999 — APPROVED** [review]
same suggestion as manish, rest looks good to me, can be merged after doing the changes.

**Pull request merged**